ext/bg/js/backend.js

```diff
@@ -786,6 +787,26 @@ class Backend {
         this._updateBadge();
     }
 
+    _onApiCreateActionPort(params, sender) {
+        let tabId;
+        if (!(
+            sender &&
+            sender.tab &&
+            (typeof (tabId = sender.tab.id)) === 'number'
```

Comment:
I wonder if we should use https://eslint.org/docs/rules/no-cond-assign. We are using strict comparisons everywhere so it's hard to mistype `=` when you actually want `===`, but it still feels like a side effect of the conditional statement.

Reply:
Also for these types of cases there's a solution in the newest browser versions: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
> Browser compatibility: Chrome 80, Firefox 74
😭

I frequently use the assign-and-test pattern elsewhere with the rationale being that it sometimes reduces lines-of-code, but I can change it here. It can be confusing, so perhaps the brevity isn't worth it.

Reply:
Style updated in b879ff83a491f0db74352eb3b789a5abe808daf2